pkg/storage/access_token_file_test.go

```diff
@@ -0,0 +1,149 @@
+package storage
+
+import (
+	storageConfig "github.com/cyberark/conjur-authn-k8s-client/pkg/storage/config"
+	. "github.com/smartystreets/goconvey/convey"
+	"os"
+	"reflect"
+	"testing"
+)
+
+type ProxyHandlerTokenFile struct {
+	AccessToken AccessTokenHandler
+}
+
+func TestAccessTokenFile(t *testing.T) {
+	var config storageConfig.Config
+	config.StoreType = storageConfig.K8S
+	config.TokenFilePath = "/tmp/accessTokenFile1"
+	var tokenInFile, _ = NewAccessTokenFile(config)
+
+	Convey("Read", t, func() {
+		Convey("Returns true if Data output as expected", func() {
+			dataActual := []byte{'t', 'e', 's', 't'}
+			err := tokenInFile.Write(dataActual)
+			So(err, ShouldEqual, nil)
+
+			dataExpected, err := tokenInFile.Read()
+			eq := reflect.DeepEqual(dataActual, dataExpected)
+
+			So(err, ShouldEqual, nil)
+			So(eq, ShouldEqual, true)
+		})
+
+		Convey("Returns error if Data is nil", func() {
+			tokenInFile.Data = nil
+			_, err := tokenInFile.Read()
+
+			So(err.Error(), ShouldEqual, "error reading access token, reason: data is empty")
+		})
+	})
+
+	Convey("Write", t, func() {
```

Comment:
Don't we also want to check if the inverse - return an error if data input is null? I believe in code we raise an error